ci-operator/step-registry/ipi/conf/nutanix/context/ipi-conf-nutanix-context-commands.sh

```diff
@@ -65,14 +65,14 @@ if [[ ! -z "${one_net_mode_network_name:-}" ]]; then
 fi
 
 subnets_json=$(curl -ks -u "${un}":"${pw}" -X POST ${api_ep} -H "Content-Type: application/json" -d @-<<<"${data}")
-subnet_uuid=$(echo "${subnets_json}" | jq ".entities[] | select (.spec.name == \"${subnet_name}\") | .metadata.uuid ")
+subnet_uuid=$(echo "${subnets_json}" | jq ".entities[1] | select (.spec.name == \"${subnet_name}\") | .metadata.uuid ")
```

Comment:
Can we confirm this change does not break the existing nutanix jobs?

Reply:
Running existing nutanix jobs passed now